ossdbtoolsservice/driver/types/mysql_driver.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, conn_params: {}, config: Optional[Configuration] = None):
             self._connection_options['port'] = constants.DEFAULT_PORT[constants.MYSQL_PROVIDER_NAME]
 
         # Setting autocommit to True initally
-        self._autocommit_status = True
+        self._connection_options['autocommit'] = True
```

Comment:
Verify by reading docs that this property dont have any concerns.
For example: commit should always work and in case of errors proper rollbacks happens.

Reply:
This doc explains it: https://dev.mysql.com/doc/refman/5.6/en/innodb-autocommit-commit-rollback.html

It should be handled properly.